src/server/scene/basic_idle_hub.cpp

```diff
@@ -133,6 +133,9 @@ ms::BasicIdleHub::~BasicIdleHub()
 void ms::BasicIdleHub::poke()
 {
     std::lock_guard lock{mutex};
+    if (!wake_lock.expired())
+        return;
```

Comment:
No braces around if block

Reply:
It may be wise to [update this.](https://mir-server.io/doc/cppguide/?showone=Conditionals#Conditionals)

Reply:
Indeed! #2416